internal/fs/fs_test.go

```diff
@@ -748,22 +748,17 @@ func TestIsDir(t *testing.T) {
 
 	for f, want := range tests {
 		got, err := IsDir(f)
-		if err != nil {
-			if want.exists != got {
-				t.Fatalf("expected %t for %s, got %t", want.exists, f, got)
-			}
-			if !want.err {
-				t.Fatalf("expected no error, got %v", err)
-			}
+		if err != nil && !want.err {
+			t.Fatalf("expected no error, got %v", err)
 		}
 
-		if got != want.exists {
+		if want.exists != got {
```

Comment:
Let's not change this and keep it consistent with the rest of the tests in the file.